module/platform/Webots/README.md

```diff
@@ -2,18 +2,21 @@ webots
 ======
 
 ## Description
-A module to connect to the Robocup controller in Webots.
+A module to connect to the Robocup controller in Webots and exchange protobuf messages with it.
```

Comment:
Technically not just the RoboCup controller, also our controllers for testing things.